app/jobs/checksum_report_job.rb

```diff
@@ -1,31 +1,28 @@
 # frozen_string_literal: true
 
-##
-# Job to run checksum report
```

Comment:
No need to document a class in two places. Move the more descriptive comment below up to serve as the class comment.